frontend/src/components/Input/index.module.css

```diff
@@ -0,0 +1,54 @@
+.input {
+  padding: 20px 24px;
+  font-family: "Work Sans", sans-serif;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 20px;
+  color: #000;
+  border: 1px solid #9f18d9;
+  border-radius: 10px;
+  box-shadow: 0 4px 5px #9f18d9 0.5;
+}
```

Comment:
Let's use the globals that Joao created.

Reply:
The input styles don't look anything like the [design](https://www.figma.com/file/36e7nnrNlYQENyigL9sQED/Wheel-of-Fortune?node-id=0%3A1)?

Reply:
@mirhamasala we are going into [hi-fi](https://www.figma.com/file/36e7nnrNlYQENyigL9sQED/Wheel-of-Fortune?node-id=496%3A2) 🥳

Reply:
Awesome. My comment still holds, though. 🙈 Maybe "the styles don't look _exactly_ like the design."

Reply:
Done